connectors/byoei.py

```diff
@@ -127,12 +135,29 @@ async def _batch_bulk(self, operations):
                         if "error" in data:
                             logger.error(f"operation {op} failed, {data['error']}")
                             raise Exception(data["error"]["reason"])
+
+            for op in operations:
+                if OP_INDEX in op or OP_UPSERT in op:
+                    self.indexed_document_count += 1
+                elif OP_DELETE in op:
+                    self.deleted_document_count += 1
+                elif "doc" in op:  # update payload
+                    self.indexed_document_volume += get_byte_size(op["doc"])
```

Comment:
let's make this new thing another PR

Reply:
Done here: https://github.com/elastic/connectors-python/pull/528